### ISSUE TITLE
Symfony Plugin breaks bootstrapFiles configuration

### ISSUE THREAD

**Issue by shyim** [phpstan/phpstan-symfony#299]
I have in my `bootstrapFiles` a file which generates the containerXml file. 
It looks like the containerXml will be parsed before the `bootstrapFiles` are executed and so the phpstan  command fails always

**Comment by ondrejmirtes** [comment]
How does it fail?

**Comment by shyim** [comment]
It just throws an error that it cannot find the xml file

**Comment by shyim** [comment]
For me it looks like that the XmlFactories are too early in the stack constructed before running the bootstrapFiles 🤔

**Comment by ondrejmirtes** [comment]
Yeah, it's not meant to be used like that, you need to have the XML file generated already when you start PHPStan. You can create an alias in Makefile or Composer scripts so that it happens every time you run PHPStan.

This could be fixed in phpstan-symfony, but you'd have to inspect the stack traces and make sure that the container isn't requested before `bootstrapFiles` are called.

**Comment by shyim** [comment]
Yes we have a script but thats annoying in combination with PhpStorm. So when you delete the whole var/cache and then change anything phpstan triggered by phpstorm explodes.

So I would rather have this fixed in the extension :D. I can deliver the stacktrace

**Comment by mitelg** [comment]
hey @ondrejmirtes 

are there plans to change this behaviour? or would a contribution be accepted?

**Comment by ondrejmirtes** [comment]
@mitelg Not sure how it could be fixed. PHPStan wants the file to already exist when you execute it. So I'd still recommend Makefile or Composer scripts.

Some people were successful by creating their own `bin/phpstan` executable that would do the things they want to do before running PHPStan and at the end it calls:

```php
Phar::loadPhar(__DIR__ . '/../vendor/phpstan/phpstan/phpstan.phar', 'phpstan.phar');

require 'phar://phpstan.phar/bin/phpstan';
```

**Comment by mitelg** [comment]
okay, I guess then this issue can be closed 👍
